lambda/models/state_machine/create_model.py

```diff
@@ -45,7 +45,8 @@ def handle_set_model_to_creating(event: Dict[str, Any], context: Any) -> Dict[st
         and request.LoadBalancerConfig  # noqa: W503
     ):
         model_table.update_item(
-            Key={"model_id": request.ModelId}, AttributeUpdates={"Status": {"Value": "CREATING", "Action": "PUT"}}
+            Key={"model_id": request.ModelId},
+            AttributeUpdates={"Status": {"Value": "CREATING", "Action": "PUT"}, "ModelConfig": {"Value": event}},
```

Comment:
I missed this line in the UpdateExpressions conversion from earlier. Is that something that can be converted easily in the next few minutes? Otherwise, I'd say let's merge as-is, then we can refactor later